.eslintrc.js

```diff
@@ -43,7 +43,6 @@ module.exports = {
           },
         ],
         curly: 'error',
-        'brace-style': 'error',
```

Comment:
CHAOS WILL ENSUE! 😂

Reply:
I had to do this, because it turned out that prettier and eslint had conflicting ideas of how braces should be handled in some cases where classes are defined. By removing this, I let prettier decide, so no need to fear chaos. 😊